torrent: wire up basic TMDB support for movies and TV

diceypotato opened the pull request:
Note: requires https://github.com/Arcadia-Solutions/arcadia-index/pull/17 with TMDB API key

FrenchGithubUser commented:
You renamed "TVShow" to "TV Show". Are you sure this is what the backend expects ?

FrenchGithubUser commented:
It should stay as "TVShow" for now, line 11 https://github.com/Arcadia-Solutions/arcadia-index/blob/main/src/models/title_group.rs

diceypotato commented:
> You renamed "TVShow" to "TV Show". Are you sure this is what the backend expects ?

Good call.  I had fixed this in the wrong place, the UI component was checking for "TV Show" to reveal the TMDB/TVDB text boxes.  Fixed it there instead.
